session_3/blogspot/app/views/layouts/application.html.erb

```diff
@@ -20,6 +22,13 @@
       <%= link_to 'Log In', login_path %>
     <% end %>
 
+    <% if current_user %>
```

Comment:
Write it as `current_user&.admin`:  https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/